auto_nag/scripts/query_creator.py

```diff
@@ -9,6 +9,13 @@
 from auto_nag.bugzilla.utils import (get_config_path, get_project_root_path,
                                      createQueriesList, cleanUp)
 
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen
+except ImportError:
+    # Fall back to Python 2's urllib2
+    from urllib2 import urlopen
+
```

Comment:
relman-auto-nag is nowhere near py3k-ready.  I'm not sure adding compat code in one place adds more value than confusion, unless there's a bigger effort to convert it?  (Same with the print below)